tests/unit/backend/corpora/common/entities/__init__.py

```diff
@@ -0,0 +1,2 @@
+def get_ids(iterable):
```

Comment:
adding a helper function this loop is used at least 24 times throughout the test code.